### DISCUSSION TITLE
ASUS USB-N13 different product id

### DISCUSSION THREAD

**Discussion by LacixHU** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#366] [Ideas]
My Asus USB-N13 has a different product id as in the plist file (it is 6385 in the plist).
In the mac report my USB-N13 has a product id of  **0x1784** (6020 in decimal)
It is not working with the current driver.
Could it be added into the next release?

**Comment by chris1111** [comment]
try to modify it yourself, tell me if it works

**Reply by LacixHU** [reply]
Could you help me how to do it?
I can modify the plist but then the Consent.command is not working. Can I modify these kext files after installation?

**Reply by LacixHU** [reply]
I've tried to modify it but the usb stick is still not recognized (not in the menu and no operating light).
Only the modified RtWlanU.kext is loaded into the memory (but the RtWlanU1827.kext is not).
